src/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -1347,30 +1351,88 @@ void CodeGen::genAVXIntrinsic(GenTreeHWIntrinsic* node)
             break;
         }
 
-        default:
-            unreached();
-            break;
-    }
+        case NI_AVX_InsertVector128:
```

Comment:
nit: I think we have these grouped by isa, then name most other places

Reply:
Thanks, will re-order.